specification/attributes/column_naming_convention.md

```diff
@@ -30,6 +30,7 @@ Naming convention for columns appearing in billing data.
 ## Exceptions
 
 * Identifiers will use the "Id" abbreviation since this is a standard pattern across the industry.
+* Product offerings that incur charges will use the "Sku" abbreviation because it is a well-understood term both within and outside the industry.
```

Comment:
nit: We should probably use the normal capitalization for the abbreviation exception and call out the ID casing separately.
```suggestion
* Product offerings that incur charges will use the "SKU" abbreviation because it is a well-understood term both within and outside the industry. When used in an ID, only the first letter will be capitalized to make it more readable (e.g., "SkuPriceId")
```

Reply:
I was following the pattern of the previous line (e.g. `Identifiers will use the "Id" abbreviation`).  Should that line change too, or does it make sense to you that the two are different?

Reply:
Per discussion on today's VS3 call, going to leave this as-is, and we will revisit.